kerchunk/intro.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Kerchunk"
+subtitle: References for (optionally) chunked, compressed n-dimensional arrays
+---
+
+## What is Kerchunk?
+
+Kerchunk is a python library for creating reference files to support cloud-optimized access to traditional geospatial file formats, like NetCDF. Kerchunk negates the need to create and store copies of data for cloud-optimized access. Given the challenge of creating and maintaining copies of data, Kerchunk is a great tool.
```

Comment:
Nobody is going to know what a `reference` file is. explain please - index file called a reference?